### ISSUE TITLE
HAProxy remote stats settings won't get applied

### ISSUE THREAD

**Issue by ikke-t** [opnsense/plugins#690]
Hi,

I tried getting some some data for my debugging of server issue. I forward all my http(s) traffic via HAProxy in OPNSense. Stuff works well, great setup in OPNSense. But when I went to HAProxy Statistics configuration page: /ui/haproxy/#subtab_haproxy-general-statistics, And did the following setup:

Stats enabled: X
Local stats TCP port: 8822
Enable remote access: X
Remote listen addresses: my opnsense LAN IP

No auth enabled, just testing here.

This should do something along this: http://cbonte.github.io/haproxy-dconv/1.8/snapshot/configuration.html#stats%20enable

but still haproxy.cfg has this for stats, and doesn't seem to listen to 8822:

    # grep -i 'stat\|8822' /usr/local/etc/haproxy.conf
        stats                       socket /var/run/haproxy.socket level admin
    # statistics are DISABLED

    # sockstat -4L | grep haproxy
    www      haproxy    52051 9  tcp4   *:443                 *:*
    www      haproxy    52051 11 tcp4   *:80                  *:*
    www      haproxy    52051 12 tcp4   *:61067               *:*

OPNSense is at the latest: 18.1.8

**Comment by ikke-t** [comment]
Hmmm... this is simpler than one can think of. User error, ooops. So I didn't realise that the "Apply" button doesn't apply it. I modified today some other settings (rules), and all the sudden the config appeared into the haproxy.cfg. So the correct topic for this issues should be:

"Apply" on HAproxy statistics page does not apply configs, only saves them.

I say saves, as I changed those values yesterday, and pressed the apply. Nothing changed. I logged in today, and did other changes, and pressed apply, and those stats settings took effect. So they were saved in config, but never applied until "Apply" hit on another page.
